azurerm/resource_arm_mariadb_server_test.go

```diff
@@ -344,6 +372,7 @@ resource "azurerm_mariadb_server" "test" {
     storage_mb            = 51200
     backup_retention_days = 7
     geo_redundant_backup  = "Disabled"
+	storage_autogrow      = "Disabled"
```

Comment:
for the Basic test could we omit this (since this is only the optional properties)